src/main/java/com/twilio/http/NetworkHttpClient.java

```diff
@@ -123,13 +126,23 @@ public Response makeRequest(final Request request) {
         }
 
         if (method == HttpMethod.POST) {
-            builder.addHeader(HttpHeaders.CONTENT_TYPE, "application/x-www-form-urlencoded");
-
-            for (Map.Entry<String, List<String>> entry : request.getPostParams().entrySet()) {
-                for (String value : entry.getValue()) {
-                    builder.addParameter(entry.getKey(), value);
+            // TODO: It will be removed after one RC Release.
+            if (request.getContentType() == null) request.setContentType(Enum.ContentType.FORM_URLENCODED);
```

Comment:
java code does not have content type set. It will throw Null pointer exception. This line of code will be removed after one RC release.